spec.html

```diff
@@ -50440,8 +50440,8 @@ <h1>Changes to FunctionDeclarationInstantiation</h1>
                 1. When the |FunctionDeclaration| _f_ is evaluated, perform the following steps in place of the |FunctionDeclaration| Evaluation algorithm provided in <emu-xref href="#sec-function-definitions-runtime-semantics-evaluation"></emu-xref>:
                   1. Let _fEnv_ be the running execution context's VariableEnvironment.
                   1. Let _bEnv_ be the running execution context's LexicalEnvironment.
-                  1. Let _fobj_ be ! _bEnv_.GetBindingValue(_F_, *false*).
-                  1. Perform ! _fEnv_.SetMutableBinding(_F_, _fobj_, *false*).
+                  1. Let _fObj_ be ! _bEnv_.GetBindingValue(_F_, *false*).
```

Comment:
I also do not like _fObj_ over _fobj_. Overall not a fan of one lowercase letter then capitalized word.